configs/recognition/tsn/activitynet/tsn_r50_320p_1x1x8_150e_anet_clip_flow.py

```diff
@@ -0,0 +1,122 @@
+# model settings
+model = dict(
+    type='Recognizer2D',
+    backbone=dict(
+        type='ResNet',
+        pretrained='modelzoo/tsn_r50_320p_1x1x8_110e_kinetics400_flow.pth',
+        depth=50,
+        in_channels=10,
+        norm_eval=False),
+    cls_head=dict(
+        type='TSNHead',
+        num_classes=200,
+        in_channels=2048,
+        spatial_type='avg',
+        consensus=dict(type='AvgConsensus', dim=1),
+        dropout_ratio=0.8,
+        init_std=0.01))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(average_clips=None)
+# dataset settings
+dataset_type = 'RawframeDataset'
+data_root = 'data/anet/flow'
+data_root_val = 'data/anet/flow'
+ann_file_train = 'data/anet/anet_train_clip.txt'
+ann_file_val = 'data/anet/anet_val_clip.txt'
+ann_file_test = 'data/anet/anet_val_clip.txt'
```

Comment:
```
data_root = 'data/ActivityNet/flow'
data_root_val = 'data/ActivityNet/flow'
ann_file_train = 'data/ActivityNet/anet_train_clip.txt'
ann_file_val = 'data/ActivityNet/anet_val_clip.txt'
ann_file_test = 'data/ActivityNet/anet_val_clip.txt'
```
To keep consist with [this](https://github.com/open-mmlab/mmaction2/blob/master/configs/localization/bmn/bmn_400x100_2x8_9e_activitynet_feature.py#L17-L22)